lib/asset_cloud/base.rb

```diff
@@ -151,6 +151,12 @@ def write(key, value)
       bucket_for(key).write(key, value)
     end
 
+    def write!(key, value)
+      asset = self[key]
+      asset.value = value
+      asset.store!
+    end
```

Comment:
Should we be logging here to stay consistent with `#write`?

Reply:
It looks like `#write` ends up being called via `Asset#store!` => `Asset#store` => `Base#write` so it should be logging already (except in the `nil` case but logging would be problematic anyway with `nil` anyway).